scripts/parse_ctest_log.rb

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/env ruby
+
+require 'getoptlong'
+require 'json'
+
+LOG_FILE_OPTION = '--log-file'
+OUTPUT_LOG_FILE_OPTION = '--output-log-file'
+ONLY_FAILED_OPTION = '--only-failed'
+HUMAN_READABLE_OPTION = '--human-readable'
+HELP_OPTION = '--help'
+
+TEST_NUMBER = 'test_number'
+TEST_NAME = 'test_name'
+TEST_SUCCESS = 'test_success'
+TESTS = 'tests'
+TESTS_COUNT = 'tests_count'
+FAILED_TESTS_COUNT = 'failed_tests_count'
+
+FAILED = 'Failed'
+PASSED = 'Passed'
+
+TIME_NOW = Time.now.strftime('[%d.%m.%Y %H:%M:%s]')
+
+opts = GetoptLong.new(
+    [LOG_FILE_OPTION, '-l', GetoptLong::REQUIRED_ARGUMENT],
+    [ONLY_FAILED_OPTION, '-f', GetoptLong::OPTIONAL_ARGUMENT],
+    [HUMAN_READABLE_OPTION, '-r', GetoptLong::OPTIONAL_ARGUMENT],
+    [OUTPUT_LOG_FILE_OPTION, '-o', GetoptLong::OPTIONAL_ARGUMENT],
+    [HELP_OPTION, '-h', GetoptLong::OPTIONAL_ARGUMENT]
+)
+
+$log_file_path = nil
+$only_failed = false
+$human_readable = false
+$output_log_file_path = nil
+
+opts.each do |opt, arg|
+  case opt
+    when LOG_FILE_OPTION
+      $log_file_path = arg
+    when ONLY_FAILED_OPTION
+      $only_failed = true
+    when HUMAN_READABLE_OPTION
+      $human_readable = true
+    when OUTPUT_LOG_FILE_OPTION
```

Comment:
This parameter is not needed - it should be removed, because we does not need a log for this script execution in a separated file.